tests/FSharp.Test.Utilities/XunitHelpers.fs

```diff
@@ -6,7 +10,13 @@ open Xunit.Abstractions
 
 open TestFramework
 
-/// Installs console support for parallel test runs.
+/// Disables custom internal parallelization added with XUNIT_EXTRAS.
```

Comment:
So, XUNIT_EXTRAS make things run as parallel as possible - but if this creates troubles, we can either undefine XUNIT_EXTRAS in the project or put this `RunTestCasesInSequence` upon a module (which is preferrable and more granular I guess).

Is this correct? Testing my understanding :)

Reply:
Yes, correct :)